### PR TITLE
Remove str instance in model of BaseException.attrs

### PR DESCRIPTION
**Pull request by mymedia2**
Do not require first exception argument to be a string. The word "usually" does not imply an obligation.
https://docs.python.org/3/library/exceptions.html#BaseException.args

## Steps

- [ ] For new features or bug fixes, add a ChangeLog entry describing what your PR does.
- [ ] Write a good description on what the PR does.

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :bug: Bug fix          |

## Related Issue

Refs: PyCQA/pylint#7233

### CODE REVIEW

**Comment by Pierre-Sassoulas** [comment]
All tests are still passing on pylint current main branch.

**Review by Pierre-Sassoulas — APPROVED** [review]
I created https://github.com/PyCQA/pylint/pull/7236 in pylint, it fails without this, it succeed with it, thank you for fixing the issue, 👍

File: astroid/interpreter/objectmodel.py [hunk 1]
```diff
@@ -672,10 +672,7 @@ def attr___dict__(self):
 class ExceptionInstanceModel(InstanceModel):
     @property
     def attr_args(self):
```
Comment (DanielNoord) [inline astroid/interpreter/objectmodel.py:674]:
```suggestion
    def attr_args(self) -> nodes.Tuple:
```

File: astroid/interpreter/objectmodel.py [hunk 2]
```diff
@@ -672,10 +672,7 @@ def attr___dict__(self):
...
-        message = node_classes.Const("")
-        args = node_classes.Tuple(parent=self._instance)
-        args.postinit((message,))
-        return args
+        return node_classes.Tuple(parent=self._instance)
```
Comment (DanielNoord) [inline astroid/interpreter/objectmodel.py:675]:
```suggestion
        return nodes.Tuple(parent=self._instance)
```

**Review by DanielNoord — CHANGES_REQUESTED** [review]
I think it makes more sense to add the test in `astroid`, that's where the change is and also is a better place for a test in case we change this later on.

@mymedia2 Could you add a test similar to the one Pierre added in this PR please?

**Comment by DanielNoord** [comment]
Superseded by https://github.com/PyCQA/astroid/pull/1749

**Pull request closed without merging**